en/docs/manage/api-rate-limit.md

```diff
@@ -0,0 +1,43 @@
+# API Rate-Limiting in Choreo
+
+Real world computation requirements are operating under finite number of computing resources. Considering this, Choreo 
```

Comment:
Shall we simplify the intro para as follows:

API rate limiting is a technique that allows you to control the rate of requests made to an API. Rate limiting helps prevent system overload and enhances API performance. When you limit the number of requests that can be made in a specific time frame, you can ensure that your API is available and responsive to all users while protecting it from malicious attacks. 

This page walks you through the steps to enable rate limiting for your APIs via Choreo and also provides information on the rate-limiting options supported by Choreo.